vcd/resource_vcd_vapp_vm.go

```diff
@@ -197,6 +197,11 @@ func resourceVcdVAppVm() *schema.Resource {
 				Default:     false,
 				Description: "Expose hardware-assisted CPU virtualization to guest OS.",
 			},
+			"properties": {
```

Comment:
For me the "properties" is a too generic name, "reserved" for some lower-level stuff.
What about "guest_properties" instead?
@dataclouder @vbauzysvmware your thoughts?

Reply:
I would also be more specific.

Reply:
`guest_properties` it is